src/EdFi.AnalyticsMiddleTier.Tests/TestCaseBase.cs

```diff
@@ -31,8 +31,11 @@ protected void PrepareTestDatabase()
             }
         }
 
+#pragma warning disable S101  // Rule: use PascalCase. But "PostgreSQL" is a proper word.
         [TestFixtureSource(typeof(DataStandardTestFixturePostgres))]
         public class TestCasePostgreSQL : TestCaseBase
+#pragma warning restore S101
```

Comment:
With "PostgreSQL" I took the opposite stance compared to "SQLServer" - that is, I left it in the proper casing. This might be a bit confusing, but here's why: "SQLServer" is not the proper name of a product; that product is properly named "Microsoft SQL Server". But "PostgreSQL" _is_ the proper name of a product, so out of respect for the proper name, I don't want to change it to "PostgreSql".

Reply:
After typing this message, the test process completed and the sarif file upload worked. Now we have a warning message above about the type name. I am surprised that we got that message, given that I have the `pragma` directive in there. I do not get this message when running `dotnet build` at the command line. I have not yet been able to find anything helpful about why CodeQL is not ignoring this suppressed error.

Note that it has the option to dismiss the alert. It would be appropriate to do that in this case. I will not click that, because I want the Analytics team to see the message. But if someone else suppresses it, then at least you'll have this screenshot below showing you what it looks like to me right now.

Reply:
![image](https://user-images.githubusercontent.com/9324390/178804594-49d603cb-8ed7-40b8-8a9b-c23309b17e6d.png)